bundles/edu.kit.ipd.sdq.pcm.simulation.scheduler.exact/src/edu/kit/ipd/sdq/pcm/simulation/scheduler/exact/factory/WindowsVistaSchedulerFactory.java

```diff
@@ -3,15 +3,17 @@
 import de.uka.ipd.sdq.scheduler.IActiveResource;
 import de.uka.ipd.sdq.scheduler.SchedulerModel;
 import de.uka.ipd.sdq.scheduler.factory.SchedulerExtensionFactory;
+import de.uka.ipd.sdq.scheduler.resources.active.IResourceTableManager;
 import edu.kit.ipd.sdq.pcm.simulation.scheduler.exact.ExactSchedulingFactory;
 
 
 public class WindowsVistaSchedulerFactory implements SchedulerExtensionFactory {
 
     @Override
-    public IActiveResource getExtensionScheduler(final SchedulerModel model, final String resourceName, final String resourceId, final long numberOfCores) {
+    public IActiveResource getExtensionScheduler(final SchedulerModel model, final String resourceName, final String resourceId, final long numberOfCores
+            , IResourceTableManager resourceTableManager) {

```

Comment:
Same

Reply:
worked in review finding; reformatted file using Palladio code formatter